src/main/java/racingCar/view/OutputView.java

```diff
@@ -0,0 +1,32 @@
+package racingCar.view;
+
+import racingCar.RacingCar;
+
+import java.util.List;
+
+public class OutputView {
+
+    private static final String HYPHEN = "-";
+    private static final String NEW_LINE = "\n";
+
+    public static void printRacingResult(){
+        System.out.println(NEW_LINE + "실행 결과");
+    }
+
+    public static void printRacing(List<RacingCar> records){
+        for (RacingCar racingCar: records) {
+            printOneCar(racingCar);
+        }
+        System.out.println();
+    }
+
+    private static void printOneCar(RacingCar car){
+        String result = "";
+        for(int i=0; i<car.positionOfRacingCar(); i++) {
```

Comment:
아직 syntax가 들쑥 날쑥 하신 부분이 있습니다 😄 
좀만 더 신경써주시면 금방 고쳐질 부분 같아요 🙏 
```suggestion
        for (int i = 0; i < car.positionOfRacingCar(); i++) {
```